Changing statistics on website

clcoffey opened the pull request:
Changing the number of individuals to 1300 from 1000 and the number of organisations from 400 to 500.

lenucksi reviewed (approved):
Great stuff, merging.

clcoffey committed:
changing statistics on website

Changing the number of individuals to 1300 from 1000 and the number of organisations from 400 to 500.

claredillon reviewed (approved):
Thanks Cristina
